EvalData/models/__init__.py

```diff
@@ -69,6 +69,15 @@
         'evaldata_pairwiseassessmentdocumenttasks',
         'evaldata_pairwiseassessmentdocumentresults',
     ),
+    (
+        'PairwiseDocumentESA',
+        PairwiseAssessmentDocumentESATask,
+        PairwiseAssessmentDocumentESAResult,
+        'pairwise-assessment-document-newui-esa',
```

Comment:
As above, let's drop "-newui" from the name.

Reply:
Changed.